app/views/shared/_page_header.html.erb

```diff
@@ -1,11 +1,13 @@
 <header>
+  <% unless caption_text === false %>
```

Comment:
Curious why triple equals over regular `==`?

Reply:
typing checking, == checks for falseyness (if thats a word), === checks it really is a false